README.md

```diff
@@ -1,8 +1,6 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
-
-
+Create two .env files: .env.test and .env.development. into each, add PGDATABASE=nc_news_test and PGDATABASE=nc_news to the respective files.
```

Comment:
The reason we don't push .env files to GitHub is so that we don't expose those variables. Please do not let people know what your databases are called.

Reply:
You can be more vague like PGDATABASE=database_name_test and PGDATABASE=database_name